src/python/pants/backend/kotlin/dependency_inference/kotlin_parser.py

```diff
@@ -144,7 +144,7 @@ async def analyze_kotlin_source_dependencies(
     source_files: SourceFiles,
 ) -> FallibleKotlinSourceDependencyAnalysisResult:
     # Use JDK 8 due to https://youtrack.jetbrains.com/issue/KTIJ-17192 and https://youtrack.jetbrains.com/issue/KT-37446.
-    request = JdkRequest("adopt:8")
+    request = JdkRequest("zulu:8.0.392")
```

Comment:
This looks a reasonable choice based on the guidance in https://whichjdk.com/#apple-silicon

Temurin is only available for java 11+.

Reply:
however, it is failing on macOS11-x86:
```
✕ src/python/pants/backend/kotlin/dependency_inference/kotlin_parser_test.py:tests failed after 3 attempts in 2.58s.
✕ src/python/pants/backend/kotlin/dependency_inference/rules_test.py:tests failed after 3 attempts in 4.30s.
```
going to see what is going on.

Reply:
The bad news is I can't reproduce these errors on my local even switching `default_version` back to `v2.1.0-M5-18-gfebf9838c` to fall back to rosetta emulation. 

The good news is that using the M1-native binary seems to make the test run almost twice as fast.